airbyte-integrations/connector-templates/destination-python/Dockerfile

```diff
@@ -1,13 +1,27 @@
-FROM python:3.7.11-alpine3.14
+FROM python:3.7.11-alpine3.14 as base
 
-# Bash is installed for more convenient debugging.
-RUN apt-get update && apt-get install -y bash && rm -rf /var/lib/apt/lists/*
+# build and load all requirements 
+FROM base as builder
+WORKDIR /airbyte/integration_code
+
+# upgrade pip to the latest version
+RUN apk --no-cache upgrade \
```

Comment:
is bash included by default? could we install it if not?

Reply:
Bash is not default Apline utility. I've added it. this is +1Mb for final size only)